src/components/NavItems.js

```diff
@@ -0,0 +1,48 @@
+import { Flex, Text, Icon, Link, Menu, MenuButton, MenuList } from '@chakra-ui/react';
+import NavHoverBox from '../components/NavHoverBox';
+import { useState } from 'react';
+
+const NavItem = ({ navSize, title, icon, description }) => {
+  const [active, setActive] = useState(false);
+  const handleSetActiveChange = () => setActive(!active);
+  return (
+    <Flex
+      mt={30}
+      flexDir="column"
+      w="100%"
+      alignItems={navSize == 'small' ? 'center' : 'flex-start'}
```

Comment:
Since I see this `navSize` repeated a lot through code, and we have the same comparison in many places (`navSize == 'small'`), we should extract the possible values for navSize to a separate file and import it everywhere:

`src/constants/index.js`
```js
export NAV_SIZE = {
    SMALL: 'small',
    LARGE: 'large',
   // .... if we need we can add more later
}
```

```js
import {NAV_SIZE} from '@constants';

....

alignItems={navSize === NAV_SIZE.small ? 'center' : 'flex-start'}

....
```